decidim-proposals/app/presenters/decidim/proposals/official_author_presenter.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    #
+    # A Null Object to abstract out the author of an official proposal.
```

Comment:
Not to be a purist, but this isn't a `null` object (as that would be an object that returns null regardless of the method call) but rather a `dummy presenter` (and not sure about this either xD).

Reply:
Either way, really good abstraction 👍

Reply:
Aaaaaahm, ok, I'll change the comment.